source/helpers/Misc.ts

```diff
@@ -7,3 +9,56 @@ export function wait(ms: number): Promise<void> {
 export function getScreenHeightProportion(proportion: number): number {
   return Math.floor((Dimensions.get("window").height * proportion) / 100);
 }
+
+export function getAPIEnvironmentIdentifierFromUrl(
+  apiUrl: string
+): string | null {
+  const apiUrlPrefixEnvironmentMapping: Record<string, string> = {
+    dev: "d",
+    release: "r",
+    staging: "s",
+    test: "t",
+  };
+
+  const prefixRegex = /^https:\/\/(?:(.*)\.)?api\.helsingborg.io/i;
+
+  const apiUrlLowercase = apiUrl.toLowerCase();
+  const match = apiUrlLowercase.match(prefixRegex);
+
+  if (match) {
+    const apiPrefix = match[1];
+
+    if (apiPrefix === undefined) {
+      return "p";
+    }
+
+    const acceptablePrefixLetter = apiUrlPrefixEnvironmentMapping[apiPrefix];
+
+    if (typeof acceptablePrefixLetter === "string") {
+      return acceptablePrefixLetter;
+    }
+  }
+  return null;
+}
+
+export function getAPIEnvironmentIdentifier(): string | null {
+  const isDev = Config.APP_ENV === "development";
```

Comment:
This detection of runtime settings is prevalent in the code base and should be refactored to a separate concern.

Reply:
Agree, however that seems a bit out of scope for this PR. I've made a new task for it instead (https://app.clickup.com/t/1tumyce).